shared/constants.ts

```diff
@@ -80,3 +80,6 @@ export const argon2iConfigForUinVid: Argon2iConfig = {
   hashLength: 5,
   mode: 'argon2i',
 };
+
+export const salt =
```

Comment:
name it agrdon2iSalt so that it's clear with name

Reply:
renamed